typescript/modules/Notification.ts

```diff
@@ -232,7 +229,13 @@ export class NotificationMenu {
     private processMessage(message: Message): JQuery | null {
         const item = $("<li>").addClass("message").data("uuid", message.uuid);
         $("<span>").addClass("message-text").html(message.message).appendTo(item);
-        $("<span>").addClass("message-close fas fa-times").appendTo(item);
+        const buttonSpan = $("<span>").addClass("message-close").appendTo(item);
```

Comment:
Why wrap everything in a `<SPAN>`, when it seems like just having an element like `<button class="message-close">` will have clearer semantics? It would also simplify the selectors used in event handling.

Reply:
Without the span, the `button` becomes the immediate child of `.dropdown-menu` and then has its size and shape distorted due to the `display: flex` rule on that element.

This wasn't obvious before because the online of the span wasn't visible, but now that it is a visible button, the result looks strange.